.Lib9c.Tests/Action/RewardGoldTest.cs

```diff
@@ -23,6 +23,8 @@ public RewardGoldTest()
             var sheets = TableSheetsImporter.ImportSheets();
             sheets[nameof(CharacterSheet)] =
                 "id,_name,size_type,elemental_type,hp,atk,def,cri,hit,spd,lv_hp,lv_atk,lv_def,lv_cri,lv_hit,lv_spd,attack_range,run_speed\n100010,전사,S,0,300,20,10,10,90,70,12,0.8,0.4,0,3.6,2.8,2,3";
+            sheets[nameof(GameConfigSheet)] =
+                "key,value\nhourglass_per_block,3\naction_point_max,120\ndaily_reward_interval,1700\ndaily_arena_interval,500\nweekly_arena_interval,56000";
```

Comment:
It can be just the flavor, you can use [*@*, a verbatim identifier](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/verbatim). There is no example of it so I leave also [stackoverflow comment](https://stackoverflow.com/questions/1100260/multiline-string-literal-in-c-sharp).

Also, it seems to import table sheets with `TableSheetsImporter.ImportSheets()`. IMO it will be better to update [.Lib9c.Tests/Data/TableCSV/GameConfigSheet.csv](https://github.com/planetarium/lib9c/blob/ee93d0ec339390de3efd8ebc1ece49fa90b7ef4e/.Lib9c.Tests/Data/TableCSV/GameConfigSheet.csv).

Reply:
I agree. Unlike `CharacterSheet`, all rows of `GameConfigSheet` should be imported, so i will use the csv data.